### ISSUE TITLE
[Bug] No effect when clicking on 'Create a new reload rule...'

### ISSUE THREAD

**Issue by pidgeon777** [blaise-io/live-reload#25]
Clicking on `Create a new reload rule...` doesn't produce any effect:

![image](https://user-images.githubusercontent.com/13739892/114673849-ec044700-9d06-11eb-9596-7a3001c7a942.png)

Chrome version: 89.0.4389.114 (Build ufficiale) (a 64 bit)

Live Reload version: 1.8.2

Windows version: 10 Pro 1803 x64

**Comment by blaise-io** [comment]
Sorry I cannot reproduce this, so I need some help for debugging the issue.

Can you follow these steps?

 - navigate to chrome://extensions
 - find the Live Reload tile, click "background page"
 - in the debug window that opens, visit the "Console" tab and monitor "All levels"
 - activate the browser window, click *Create a new reload rule...*
 - activate the debug window and copy all contents
 - paste it in this issue

**Comment by pidgeon777** [comment]
Hi, I disabled every extension except yours and the **Extensity** one (for managing the extensions, just disable/enable them).

Tested everything on this very same github page.

Log:

````
background.js:122 3150 "Navigated to new URL:" "https://github.com/blaise-io/live-reload/issues/25"
background.js:249 3150 "disable tab monitoring"
background.js:133 Incoming message from  chrome-extension://jcejoncdonagmfohjcdgohnmecaipidc/popup.html {type: "isMonitoring?"}
````

**Comment by blaise-io** [comment]
I will check a few thing. Note to self: [chrome.window.open](https://github.com/blaise-io/live-reload/blob/master/app/popup/popup.ts#L88) seems to fail.

Workaround:  Right click "Create a new reload rule..." and open in new tab work

<img width="250" src="https://user-images.githubusercontent.com/788368/116777998-cbaaeb00-aa6f-11eb-9b97-dfa70027322f.png">

Or visit chrome-extension://jcejoncdonagmfohjcdgohnmecaipidc/form.html directly

**Comment by blaise-io** [comment]
Can you try on 1.8.3? May be fixed by fbfce2400be43dc82ee33fb0baf04891045341bf
